client/package.json

```diff
@@ -10,8 +10,17 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@chakra-ui/icons": "^2.1.1",
+    "@chakra-ui/react": "^2.8.1",
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@tanstack/react-table": "^8.10.0",
+    "@tanstack/table-core": "^8.10.0",
+    "antd": "^5.9.2",
```

Comment:
What the use of antd in this. Remove unwanted libraries

Reply:
Done.Removed unwanted librries